eventHandlers.js

```diff
@@ -0,0 +1,55 @@
+var CarLot = (function(oldCarLot){
+
+	var selectedId = "";
+	var userInput = document.getElementById("textInputBar");
+
+	oldCarLot.activateEvents = function (array) {
+		var inventoryCards = document.getElementsByClassName("inventoryCard");
+		var textInputBar = document.getElementById("textInputBar");
```

Comment:
You're grabbing ```textInputBar``` here but you've already grabbed it on a higher scope on line 4, so you could just use that one internally to this function as well (in fact, you don't even use ```textInputBar``` in here)